build/buildEnvironment.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Version
 export AWIPSII_VERSION="20.3.2"
-export AWIPSII_RELEASE="0.1"
+export AWIPSII_RELEASE="0.2"
```

Comment:
Don't know if it's a big deal, but I'll need to update this file on linux to be 0.3

Reply:
Okay, ya the mac doesn't look there at all, so I think it's fine to build it like this?